packages/idyll-components/src/cite.js

```diff
@@ -0,0 +1,79 @@
+const React = require('react');
+
+const citations = [];
+const referenceInstances = [];
+
+const citationsChangeHandler = {
+  set: function(target, property, value) {
+    referenceInstances.forEach(referenceInstance => {
+      referenceInstance.forceUpdate();
+    });
+    target[property] = value;
+    return true;
+  }
+};
+
+var citationsProxy = new Proxy(citations, citationsChangeHandler);
+
+class References extends React.Component {
+  constructor() {
+    super();
+    referenceInstances.push(this);
+  }
+  render() {
+    return (
+      <div id="references">
+        <h1>References</h1>
+        <ol>
+          {citations.map((citation, index) => (
+            <li key={index}>
+              {`${citation.authors}: `}
```

Comment:
Can this be simplified to `{citation.authors}:` or is this a whitespace preservation thing?

Reply:
Replacing it with this to preserve whitespace
```
 {citation.authors}:{' '}
     <a href={citation.url} target="_blank">
```
Or should i remove it all together?

Reply:
That works 👍

Reply:
Done